README.md

```diff
@@ -21,7 +21,14 @@ See [picturingjordan.com](https://picturingjordan.com) for an example of this th
 - configurable keywords for every post
 
 ## Usage
-Clone the repository to your site's `themes` directory. Refer to [`exampleSite/config.toml`](https://github.com/alanorth/hugo-theme-bootstrap4-blog/blob/master/exampleSite/config.toml) for recommended configuration values.
+Clone the repository to your site's `themes` directory.
+
+### If you already using git
+git submodule add https://github.com/alanorth/hugo-theme-bootstrap4-blog.git themes/hugo-theme-bootstrap4-blog
+hugo-theme-bootstrap4-blog
+
+### Start with
```

Comment:
I don't think we need a new section for this. The general case is to clone the theme to your "themes" folder and then to reference the example config. The "If You're Already Using git" section is just an extra case for advanced users and shouldn't change the flow of the documentation.